pootle/core/views.py

```diff
@@ -74,9 +74,11 @@ def set_permissions(f):
     @functools.wraps(f)
     def method_wrapper(self, request, *args, **kwargs):
         User = get_user_model()
-        request.profile = User.get(self.request.user)
```

Comment:
hmm... I checked now and I understand your comment
> user.is_anonymous() makes no DB calls and user.is_authenticated()
so I'm not sure now if this change was necessary. I'll check this in `pdb`.